public/js/graph/react_graph.js

```diff
@@ -438,6 +440,34 @@ var Graph = React.createClass({
         });
     },
 
+    onButtonPress: function(zoomOrPanFunction, direction) {
+        this.setState({mouseDown: setInterval(() => zoomOrPanFunction(direction), 100)});
```

Comment:
Also, before this line, you should call `zoomOrPanFunction(direction)` once so that if a user clicks (but doesn't hold) the button, it'll still work.